receiver/collectdreceiver/collectd.go

```diff
@@ -234,9 +234,7 @@ func labelsFromName(val *string) (instanceName string, toAddDims map[string]stri
 				}
 				piece := dimensions[prev:cindex]
 				tindex := strings.Index(piece, "=")
-				//lint:ignore S1004 staticcheck wants us to use strings.Contains instead of Index but we need to use
```

Comment:
@owais this was the intention of my comment at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/44#discussion_r353546040